lms/static/js/courseware/base/views/proctored_exam_view.js

```diff
@@ -0,0 +1,42 @@
+;(function (define) {
+
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'gettext'
+], function ($, _, Backbone, gettext) {
+
+   'use strict';
+
+    return Backbone.View.extend({
+        initialize: function (options) {
+            this.$el = options.el;
+            this.model = options.model;
+            this.templateId = options.proctored_template;
+            this.template = _.template($(this.templateId).html());
+            this.timerId = null;
+        },
+
+        render: function () {
+            var html = this.template(this.model.toJSON());
```

Comment:
Just added some guards for this. I'll see if it fixes the test failures on the build.